### PR TITLE
disable installation of bundled BioPerl and HTSLib if they are dependencies of VEP

### PR DESCRIPTION
**Pull request by lexming**
Currently VEP installs a bundled BioPerl regardless of the dependency in EB

```
 Setting up directories
   - fetching BioPerl
   - unpacking /apps/brussel/CO7/skylake/software/VEP/96.0-foss-2019a-Perl-5.28.1/modules/api/Bio/tmp/release-1-6-924.zip
```

This PR disables the installation of the bundled BioPerl if it is present in the dependency list of VEP. Same logic is used for HTSLib, which is now always disabled.

### CODE REVIEW

File: easybuild/easyblocks/v/vep.py [hunk 1]
```diff
@@ -69,13 +69,27 @@ def install_step(self):
...
+        installopt_deps = ' '.join(installopt_deps)
+
         # see https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#installer
         cmd = ' '.join([
             self.cfg['preinstallopts'],
             'perl',
             'INSTALL.pl',
-            # don't try to install optional Bio::DB::HTS (can be provided as an extension instead)
-            '--NO_HTSLIB',
+            # disable any bundled dependencies provided by EB
```
Comment (smoors) [inline easybuild/easyblocks/v/vep.py:91]:
```suggestion
            # disable installation of bundled dependencies that are provided as dependencies in the easyconfig
```

**Review by smoors — CHANGES_REQUESTED** [review]
looks good to me, apart from a comment update.
can you also upload a test report?

**Comment by lexming** [comment]
Test report by @lexming
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _VEP-94.0-foss-2018b-Perl-5.28.0.eb_ 
 * **SUCCESS** _VEP-94.5-foss-2017b-Perl-5.26.0.eb_ 
 * **SUCCESS** _VEP-95.0-foss-2018b-Perl-5.28.0.eb_ 
 * **SUCCESS** _VEP-96.0-foss-2019a-Perl-5.28.1.eb_ 

Build succeeded for 4 out of 4 (4 easyconfigs in total)
node354.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f07d9529fba7a58c4958661274e45c5b for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _VEP-94.0-foss-2018b-Perl-5.28.0.eb_ 
 * **SUCCESS** _VEP-94.5-foss-2017b-Perl-5.26.0.eb_ 
 * **SUCCESS** _VEP-95.0-foss-2018b-Perl-5.28.0.eb_ 
 * **SUCCESS** _VEP-96.0-foss-2019a-Perl-5.28.1.eb_ 
 * **SUCCESS** _VEP-103.1-GCC-10.2.0.eb_ 

Build succeeded for 5 out of 5 (5 easyconfigs in total)
node354.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4aad35e9ded85a13bfa4ab3d90dcd3ec for a full test report.

**Comment by lexming** [comment]
@smoors tasty warm tests out of the oven :slightly_smiling_face:

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**